assets/admin/students/student-modal/index.js

```diff
@@ -71,6 +71,7 @@ const POSSIBLE_ACTIONS = {
  * @param {Function} props.onClose  Close callback.
  * @param {Array}    props.students A list of Student ids related to the action should be applied.
  */
+
```

Comment:
I think we don't need this empty line here?

Reply:
[here ](https://github.com/Automattic/sensei/pull/5004/commits/1d9606ce104954454b3640cc6433b11a2ebab6dd)